src/plugins/unified_search/public/saved_query_management/saved_query_management_list.tsx

```diff
@@ -227,96 +323,179 @@ export function SavedQueryManagementList({
         data: {
           attributes: savedQuery.attributes,
         },
-        append: !!showSaveQuery && (
-          <EuiButtonIcon
-            css={css`
-              opacity: 0.2;
-              filter: grayscale(100%);
-
-              &:hover,
-              &:focus:focus-visible {
-                opacity: 1;
-                filter: grayscale(0%);
-              }
-            `}
-            iconType="trash"
-            aria-label={`Delete ${savedQuery.attributes.title}`}
-            data-test-subj={`delete-saved-query-${savedQuery.attributes.title}-button`}
-            title={`Delete ${savedQuery.attributes.title}`}
-            onClick={() => handleDelete(savedQuery)}
-            color="danger"
-          />
-        ),
       };
-    }) as unknown as SelectableProps[];
-  };
+    });
+  }, [currentPageQueries, format, selectedSavedQuery]);
 
-  const renderOption = (option: RenderOptionProps) => {
-    return <>{option.attributes ? itemLabel(option.attributes) : option.label}</>;
-  };
+  const renderOption = useCallback(
+    (option: RenderOptionProps) => {
+      return (
+        <>
+          {option.attributes ? itemLabel(option.attributes) : option.label}
+          {option.value === loadedSavedQuery?.id && (
+            <EuiBadge color="hollow" css={{ marginLeft: euiThemeVars.euiSizeS }}>
+              {i18n.translate('unifiedSearch.search.searchBar.savedQueryActiveBadgeText', {
+                defaultMessage: 'Active',
+              })}
+            </EuiBadge>
+          )}
+        </>
+      );
+    },
+    [loadedSavedQuery?.id]
+  );
 
-  const canEditSavedObjects = application.capabilities.savedObjectsManagement.edit;
+  const countDisplay = useMemo(() => {
+    const parts = [
+      i18n.translate('unifiedSearch.search.searchBar.savedQueryTotalQueryCount', {
+        defaultMessage: '{totalQueryCount, plural, one {# query} other {# queries}}',
+        values: { totalQueryCount },
+      }),
+    ];
+
+    if (Boolean(selectedSavedQuery)) {
+      parts.push(
+        i18n.translate('unifiedSearch.search.searchBar.savedQuerySelectedQueryCount', {
+          defaultMessage: '1 selected',
```

Comment:
This seems a little odd for the UX here since it's not possible to select more than one, and as soon as you select one you can't deselect it. I think it might make more sense to just remove this text, but I don't have a super strong opinion on it.

Reply:
@stratoula In this comment thread we decided to leave it for now until we figure out how to improve the overall UX around selected queries: https://github.com/elastic/kibana/pull/170599#discussion_r1464725921. I created a followup issue here to discuss and address some of the outstanding concerns so that in the meantime we can get the important fixes this PR addresses in to 8.13: #176730.